app/components/Meetup.js

```diff
@@ -1,37 +1,13 @@
 import React from 'react';
 
 class Meetup extends React.Component {
```

Comment:
@natevenn To stay consistent with the other components, should we use the following for imports and class declarations:

import React, {Component} from 'react';
export default class Meetup extends Component